### ISSUE TITLE
Pangolin install missing `sudo git submodule update --init`

### ISSUE THREAD

**Issue by adthoms** [MarineRoboticsGroup/cora#25]
Commands to install Pangolin should include `sudo git submodule update --init` as follows:

```bash
# install Pangolin
git clone git@github.com:stevenlovegrove/Pangolin.git
cd Pangolin
sudo git submodule update --init
git checkout d484494 # this is the most recent commit that worked for everything
mkdir build
cd build
cmake ..
make -j
sudo make install
```

**Comment by alanpapalia** [comment]
nice catch! Thanks!

**Comment by alanpapalia** [comment]
closed with [242b4dc](https://github.com/MarineRoboticsGroup/cora/commit/242b4dc753a8a0b500e5d2a6bdf81a04e35e6f68)
